LoopKitUI/Extensions/Guardrail+UI.swift

```diff
@@ -12,16 +12,19 @@ import LoopKit
 
 
 extension Guardrail where Value == HKQuantity {
-    func color(for quantity: HKQuantity) -> Color {
+    func color(for quantity: HKQuantity,
+               inRangeColor inRange: Color = .primary,
+               warningColor warning: Color = .warning,
+               criticalColor critical: Color = .critical) -> Color {
```

Comment:
[nit] I know these colors get used together a lot, so I'm wondering if it could make sense to have a `struct` that could contain all 3.

Reply:
I'm not against it, but I remember we talked about having a colour palette but opted to just using individual colours. Happy to add it if other agree with @novalegra.

Reply:
These could make sense to bundle, I think.